src/valkey-benchmark.c

```diff
@@ -1583,7 +1583,7 @@ int parseOptions(int argc, char **argv) {
         "                    the scheme 'valkeys'.\n"
         " -c <clients>       Number of parallel connections (default 50).\n"
         "                    Note: If --cluster is used then number of clients has to be\n"
-        "                    the same or higher than the number of nodes.\n"
+        "                    the same or greater than the number of nodes.\n"
```

Comment:
I find that `higher than` should generally be replaced by `greater than`, `more than`, or `above`. With `greater` generally being the right replacement. But I'm happy to drop any such corrections.